smallrye-reactive-messaging-provider/revapi.json

```diff
@@ -0,0 +1,85 @@
+[
+    {
+        "extension": "revapi.java",
+        "id": "java",
+        "configuration": {
+            "missing-classes": {
+                "behavior": "report",
+                "ignoreMissingAnnotations": false
+            }
+        }
+    },
+    {
+        "extension": "revapi.filter",
+        "configuration": {
+            "elements": {
+                "include": [
+                    {
+                        "matcher": "java-package",
+                        "match": "io.smallrye.reactive.messaging.providers"
```

Comment:
I would have excluded the provider completely. 
Public API should be in the API module.

Reply:
Originally it was there for `PublisherDecorator`, but I can reference it directly. 
The idea would be to at least move and/or extract interface for classes like `PublisherDecorator` and `ExecutionHolder`, `WorkerPoolRegistry`, `ContextAwareMessage`, `LocalContextMetadata`.